src/core/capture_loop.py

```
@@ -5,11 +5,15 @@
 """
 import logging
 import time
-from typing import Optional
+import random
+from typing import Optional, List
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)